### ISSUE TITLE
roberta-large LSTM 결과

### ISSUE THREAD

**Issue by Trailblazer-Yoo** [BoocampMonster/korean-relation-extraction#13]
<img width="898" alt="스크린샷 2022-11-27 오전 9 47 52" src="https://user-images.githubusercontent.com/97590480/204114284-aef08cff-8078-44c0-afec-de1717784c33.png">

<img width="777" alt="스크린샷 2022-11-27 오전 9 49 27" src="https://user-images.githubusercontent.com/97590480/204114330-ed962e92-a939-4628-84cb-32321d04d6aa.png">

@hundredeuk2 
이렇게 LSTM 모듈을 roberta-large 위에 추가해서 돌렸는데 아예 f1이 박살났는데??
regressor 구성이 달라서 그런건지 혹시 차이점이나 아니면 이게 정상인지 확인좀

**Comment by hundredeuk2** [comment]
> 

nn Cell이 biLSTM이 있고 LSTM이 있는게 어떤 차이인지 확인하고 아려줘볼께 일단 용훈이가 예전에 해준거랑 내가 한건 둘다 biLSTM과 biGRU cell이야

글구 regressor가 다르기도 해

**Comment by hundredeuk2** [comment]
미안 이전에 달아준건 내가 잘못이해 하고 달았다. 

지금 짠 코드에 의하면 GRU hidden state랑 linear 랑 비선형관계가 아니라 선형관계라 정보반영이 안된거 같아 그래서 GRU ouput을 먼저 Activation을 넣어주고 다른 함수에 넣어줘야 비선형접근이 되는 거같아!

**Comment by hundredeuk2** [comment]
`self.regressor = nn.Sequential(
            nn.ReLU()
            nn.Dropout(self.drop_rate),
            nn.Linear(self.model.config.hidden_size*2, num_labels))`
가 되야 하지 않을까 싶네

**Comment by Trailblazer-Yoo** [comment]
맞아 깜빡하고 저걸로 비슷하게 Tanh으로 돌렸는데도 박살나더라.. 내일 같이 한번 확인점 해보자

**Comment by Trailblazer-Yoo** [comment]
Solve) roberta 모델은 학습률을 1-e5로 해야 한다.(내가 준 학습률은 5-e5)

<img width="1083" alt="스크린샷 2022-11-26 오후 11 15 17" src="https://user-images.githubusercontent.com/97590480/204196864-8b755a21-d8c2-4bfd-af1b-0442328cb13a.png">

<img width="910" alt="스크린샷 2022-11-28 오후 1 57 41" src="https://user-images.githubusercontent.com/97590480/204197250-03fb1df5-0213-49b9-9b01-c5f986d3b6b7.png">

roberta를 사전학습 시킬때는 위의 learning rate를 가지지만 파인 튜닝을 할 때에는 좀더 낮은 learning rate로 사전학습 모델의 파라미터를 최대한 살리는 방식으로 진행해야 함
